frontend/src/component/main/modelOption/ModelOption.tsx

```diff
@@ -0,0 +1,92 @@
+import {flexBetween, flexCenter} from '@/style/common';
+import {Title4_Medium} from '@/style/fonts';
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
+import Icon from '@/component/common/icons';
+import ModelOptionGroup from './ModelOptiongroup';
+
+import mainOption from '@/assets/mocks/mainOption.json';
+import outerOption from '@/assets/mocks/outerColorOption.json';
+
+type Option = {
+  icon: string;
+  name: string;
+};
+
+type ExclusiveOption = {
+  exclusive: Option[];
+  LeBlanc: Option[];
+  Prestige: Option[];
+  Calligraphy: Option[];
+};
+
+function ModelOption({name}: {name: string}) {
+  const [optionData, setOptionData] = useState<ExclusiveOption | null>(null);
+
+  useEffect(() => {
+    let selectedData = null;
+
+    switch (name) {
+      case '핵심 옵션':
+        selectedData = mainOption;
+        break;
+      case '외장 색상':
+        selectedData = outerOption;
+        break;
+      case '내장 색상':
+        selectedData = mainOption;
+        break;
+    }
+    if (selectedData) {
+      setOptionData(selectedData);
+    }
```

Comment:
switch 대신 객체로 관리하면 어떨까요?
```suggestion
const optionType = {
'핵심 옵션' : mainOption,
'외장 색상' : outerOption,
'내장 색상' : mainOption,
}

setOptionData(optionType[name]);

```